xconf-angular-admin/src/main/webapp/app/xconf/config/state.config.js

```diff
@@ -329,21 +329,21 @@
                 }
             })
             .state('telemetrytwoprofile-edit', {
-                controller: 'TelmetryTwoProfileEditController',
+                controller: 'TelemetryTwoProfileEditController',
                 controllerAs: 'vm',
                 url: '/telemetrytwoprofile/edit/:telemetryProfileId',
                 templateUrl: 'app/xconf/telemetry/telemetrytwoprofile/telemetrytwoprofile.edit.html',
                 data: {
                     permissions: [PERMISSION.WRITE_TELEMETRY_ALL, PERMISSION.WRITE_TELEMETRY_STB, PERMISSION.WRITE_TELEMETRY_XHOME, PERMISSION.WRITE_TELEMETRY_RDKCLOUD]
                 }
             })
-            .state('telemetrytwoprofile-view', {
-                controller: 'TelmetryTwoProfileViewController',
+            .state('telemetrytwoprofile-import', {
+                controller: 'TelemetryTwoProfileImportController',
                 controllerAs: 'vm',
-                url: '/telemetrytwoprofile/view/:telemetryProfileId',
-                templateUrl: 'app/xconf/telemetry/telemetrytwoprofile/telemetrytwoprofile.view.html',
+                url: '/telemetrytwoprofile/import',
+                templateUrl: 'app/xconf/telemetry/telemetrytwoprofile/telemetrytwoprofile.import.html',
                 data: {
-                    permissions: [PERMISSION.READ_TELEMETRY_ALL, PERMISSION.READ_TELEMETRY_STB, PERMISSION.READ_TELEMETRY_XHOME, PERMISSION.READ_TELEMETRY_RDKCLOUD]
+                    permissions: [PERMISSION.WRITE_TELEMETRY_ALL, PERMISSION.WRITE_TELEMETRY_STB, PERMISSION.WRITE_TELEMETRY_XHOME, PERMISSION.WRITE_TELEMETRY_RDKCLOUD]

```

Comment:
Use PERMISSION.WRITE_TELEMETRY_PERMISSIONS here to include sky permissions as well.